flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -140,6 +175,84 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
     }
   }
 
+  private fun fetchNativeFrames(result: Result) {
+    if (!isFrameMetricsAggregatorEnabled) {
+      result.error("1", "Native frames not available.", null)
+      return
+    }
+
+    try {
+      var totalFrames = 0
```

Comment:
What if we make `ActivityFramesTracker` (see Android repo) public but internal, and rather use it? DRY

Reply:
https://github.com/getsentry/sentry-java/pull/1931